src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/DefaultOpenDistroSecurityKeyStore.java

```diff
@@ -950,4 +976,36 @@ private static void checkPath(String keystoreFilePath, String fileNameLogOnly) {
                 + fileNameLogOnly);
         }
     }
+
+    private List<String> getOtherName(List<?> altName) {
+        ASN1Primitive oct =  null;
+        try {
+            byte[] altNameBytes = (byte[]) altName.get(1);
+            oct = (new ASN1InputStream(new ByteArrayInputStream(altNameBytes)).readObject());
+        } catch (IOException e) {
+            throw new RuntimeException("Could not read ASN1InputStream", e);
+        }
+        if (oct instanceof ASN1TaggedObject) {
+            oct = ((ASN1TaggedObject) oct).getObject();
+        }
+        ASN1Sequence seq = ASN1Sequence.getInstance(oct);
+
+        // Get object identifier from first in sequence
+        ASN1ObjectIdentifier asnOID = (ASN1ObjectIdentifier) seq.getObjectAt(0);
+        String oid = asnOID.getId();
+
+        // Get value of object from second element
+        final ASN1TaggedObject obj = (ASN1TaggedObject) seq.getObjectAt(1);
+        // Could be tagged twice due to bug in java cert.getSubjectAltName
+        ASN1Primitive prim = obj.getObject();
```

Comment:
slightly confused by what this code section does, can you mention the bug you're referencing.